fixtures/cassettes/collections_requests.yml

```diff
@@ -2,6 +2,19 @@
 http_interactions:
 - request:
     method: get
+    uri: https://cmr.earthdata.nasa.gov/search/collections.json?echo_collection_id%5B%5D=C179003030-ORNL_DAAC&include_has_granules=true&include_granule_counts=true
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      User-Agent:
+      - Faraday v0.8.8
+      Client-Id:
+      - eed-edsc-dev
```

Comment:
Not sure why it hasn't ever record this request. I checked the ```edsc_configurer.rb``` - we don't match_request_on client-id in this situation. I also changed the client_id to something else and ran the test, no new fixtures were added.